### PR TITLE
replaces the engineering outpost on icemoon with a new ruin, unlucky engineer.

### PR DESCRIPTION
**Pull request by CoiledLamb**
## About The Pull Request
still wip. i made an ice themed engineering ruin and then realized there was a kind of dated one already on icebox. so i decided to rotate it out in favor of a new ruin, Unlucky Engineer.
a heavily damaged engineering outpost, that's still repairable, but infested by fauna. surviving here will take your wits, and the payoff is a nice increase to the power grid - as well as many other engineering themed goodies.
also allows mappers to set the lifetime of legion broods to any value, so you can place them down as easy to kill mobs. needed to use this here, but i hope it comes in handy.
old ruin:
image
new ruin: 

## Why It's Good For The Game
more interesting ruin design, implements a lot of helpers, decals, and mobs we didn't have years ago. makes be best loot a bit harder to get by putting obstacles in the way of it in the form of more mobs
## Changelog
:cl:
add: replaces engineering outpost on icemoon with unlucky engineer
/:cl:

### CODE REVIEW

**Review by Maurukas — CHANGES_REQUESTED** [review]
First pass review for issues mentioned in mapping general.

* You have an active turf problem, primarily due to the use of `/airless` turf subtypes instead of `/icemoon` subtypes. This causes them to spawn with a vacuum on them at roundstart. You can use the in-game mapping tools to track them down.
* Please remember to delete `_maps/RandomRuins/IceRuins/icemoon_surface_engioutpost.dmm`, as you're deleting it's map datum and replacing it with a new file.

File: code/datums/ruins/icemoon.dm [hunk 1]
```diff
@@ -67,6 +61,13 @@
 	description = "Here lies Charles Morlbaro. He died the way he lived."
 	suffix = "icemoon_surface_smoking_room.dmm"
 
+/datum/map_template/ruin/icemoon/underground/unlucky_engineer
+	name = "Ruined Engineering Outpost"
+	id = "unlucky_engineer"
+	description = "A bloody hand reaches for a ruined APC. Is whoever - or whatever destroyed this place still lurking in its shadows?"
+	suffix = "icemoon_unlucky_engineer.dmm"
+	always_place = TRUE
```
[the remark this answers is not in the document]
Comment (CoiledLamb) [reply code/datums/ruins/icemoon.dm:69]: 👍 will do

**Comment by mc-oofert** [comment]
image
among us

File: code/game/area/areas/ruins/icemoon.dm [hunk 2]
```diff
@@ -42,9 +42,6 @@
 /area/ruin/pizzeria/kitchen
 	name = "\improper Moffuchi's Kitchen"
 
-/area/ruin/planetengi
-	name = "\improper Engineering Outpost"
-
```
Comment (Maurukas) [inline code/game/area/areas/ruins/icemoon.dm:47]: This area still exists somewhere and causing CI failures. Check your map files.

**Review by Maurukas — CHANGES_REQUESTED** [review]

**Comment by CoiledLamb** [comment]
i'm doing the caves of qud mod jam this week so i'll fix this one up when that finishes

**Comment by san7890** [comment]
this staled out three separate times just make a new PR when you're ready again

**Comment by Addust** [comment]
that's just syndicate lavaland engineering

**Pull request closed without merging**